src/ui/component/commentsList/view.jsx

```diff
@@ -0,0 +1,86 @@
+// @flow
+import * as React from 'react';
+import relativeDate from 'tiny-relative-date';
+
+type CommentListProps = {
+  comments: Array<any>,
+  fetchList: string => void,
+  uri: string,
+  isLoading: boolean,
+};
+
+type CommentProps = {
+  commentId: string,
+  claimId: string,
+  author: string,
+  message: string,
+  parentId: number,
+  timePosted: number,
+};
+
+class CommentList extends React.PureComponent<CommentListProps> {
+  componentDidMount() {
+    this.fetchComments(this.props);
+  }
+
+  fetchComments = (props: CommentListProps) => {
+    const { fetchList, uri } = props;
+    fetchList(uri);
+  };
+
+  render() {
+    const { comments } = this.props;
+
+    if (!comments) {
+      return null;
+    }
+
+    return (
+      <section>
+        <ul className="comments">
+          {comments.map(comment => {
+            return (
+              <Comment
+                author={comment.channel_name}
+                claimId={comment.channel_id}
+                commentId={comment.comment_id}
+                key={comment.channel_id + comment.comment_id}
+                message={comment.comment}
+                parentId={comment.parent_id || null}
+                timePosted={comment.timestamp * 1000}
+              />
+            );
+          })}
+        </ul>
+      </section>
+    );
+  }
+}
+
+class Comment extends React.PureComponent<CommentProps> {
```

Comment:
Comment should be broken into it's own file. Then in CommentsList, pass `commentId` that can be used to select the comment. 

Then if we ever require more data to display a comment, we only have to update one place, instead of having to pass another prop to it in this file